packages/ui/src/components/elements/Button/Button.tsx

```diff
@@ -21,9 +22,10 @@ const Button = ({
         variant,
         className
       )}
+      disabled={disabled || loading} // button disabled when loading is true. 
       {...htmlButtonProps}
     >
-      {label}
+      {loading ? <i className="fa fa-spinner fa-spin"></i> : label}
```

Comment:
Use the `Loader` component that's already been built.

Reply:
done now. :)